<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1001</width>
      <height>561</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>262</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuColor">
    <property name="title">
     <string>Color</string>
    </property>
    <addaction name="actionRed"/>
    <addaction name="actionGreen"/>
    <addaction name="actionBlue"/>
   </widget>
   <widget class="QMenu" name="menuWidth">
    <property name="title">
     <string>Width</string>
    </property>
    <addaction name="action3"/>
    <addaction name="action6"/>
    <addaction name="action9"/>
    <addaction name="action12"/>
   </widget>
   <widget class="QMenu" name="menuFill_color">
    <property name="title">
     <string>Fill color</string>
    </property>
    <addaction name="actionRed_2"/>
    <addaction name="actionGreen_2"/>
    <addaction name="actionBlue_2"/>
   </widget>
   <widget class="QMenu" name="menuFigure">
    <property name="title">
     <string>Figure</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionPolygon"/>
    <addaction name="actionPolyline"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuColor"/>
   <addaction name="menuFill_color"/>
   <addaction name="menuWidth"/>
   <addaction name="menuFigure"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action9">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="actionRed_2">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionGreen_2">
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionBlue_2">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
  <action name="actionPolyline">
   <property name="text">
    <string>Polyline</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
